app/app-controllers/error-screen/errorController.js

```diff
@@ -67,12 +69,12 @@ ErrorController.prototype._removeModalEvents = function() {
 };
 
 ErrorController.prototype._generateErrorCallback = function(params) {
-    var self = this;
-    var navigator = params.navigator;
-    var backHandler = params.backHandler;
-    var retryHandler = params.retryHandler;
-    var isActiveItem = params.isActiveItem;
-    var canGoBack = params.canGoBack;
+    const self = this;
```

Comment:
Just figured this out. Yes, we need `self` here because we aren't, and can't, use an arrow function below in the function we return out of this call.